Simulator/tft_simulator.py

```diff
@@ -191,6 +191,9 @@ def step(self, action):
                 # Take a game action and reset actions taken
                 self.actions_taken = 0
                 self.game_round.play_game_round()
+                self.observations = {agent: self.game_observations[agent].get_lobo_observation(self.PLAYERS[agent],
```

Comment:
@silverlight6 Important to generate the observation again after the play_game_round